au4/src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -181,6 +200,32 @@ void TrackeditActionsController::paste()
     }
 }
 
+void TrackeditActionsController::clipSplit(const ActionData& args)
+{
+    ClipKey clipKey = args.arg<ClipKey>(0);
+    if (!clipKey.isValid()) {
+        return;
+    }
+
+    secs_t playbackPosition = globalContext()->playbackState()->playbackPosition();
+
+    trackeditInteraction()->split(clipKey.trackId, playbackPosition);
+
+    auto trackeditProject = globalContext()->currentProject()->trackeditProject();
+    trackeditProject->pushHistoryState("Split", "Split");
+}
+
+void TrackeditActionsController::trackSplit()
+{
+    TrackId selectedTrackId = selectionController()->selectedTrack();
```

Comment:
we need to either add the track id argument here, or send a new action with the track id (as is done for clips in doGlobalSplit)